lib/connection_pool/connection_manager.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'connection_wrapper'
+
+
+class ConnectionPool::ConnectionManager
```

Comment:
Looking how it used I think this class is not necessary here. Enough to have getters and setters on ConnectionPool class.